templates/repo/actions/runs_list.tmpl

```diff
@@ -6,31 +6,35 @@
 	</div>
 	{{end}}
 	{{range .Runs}}
-		<li class="item gt-df gt-py-4">
+		<li class="item gt-df gt-py-3">
 			<div class="issue-item-left issue-item-icon gt-df gt-items-start">
 				{{template "repo/actions/status" (dict "status" .Status.String "locale" $.locale)}}
 			</div>
-			<div class="issue-item-main action-item-main gt-f1 gt-fc gt-df gt-mr-3">
+			<div class="issue-item-main action-item-main gt-f1 gt-fc gt-df gt-mr-3 gt-gap-2">
 				<div class="issue-item-top-row gt-mb-2">
 					<a class="index gt-no-underline title action-item-title" title="{{.Title}}" href="{{if .Link}}{{.Link}}{{else}}{{$.Link}}/{{.Index}}{{end}}">
 						{{- .Title -}}
 					</a>
 				</div>
 				<div class="desc issue-item-bottom-row gt-df gt-ac gt-fw">
-					<b>{{if not $.CurWorkflow}}{{.WorkflowID}} {{end}}#{{.Index}}</b>: {{$.locale.Tr "actions.runs.commit"}}
-					<a href="{{$.RepoLink}}/commit/{{.CommitSHA}}">{{ShortSha .CommitSHA}}</a>{{$.locale.Tr "actions.runs.pushed_by"}} {{.TriggerUser.GetDisplayName}}
-					<span class="ui label">
-						{{if .RefLink}}
-							<a href="{{.RefLink}}">{{.PrettyRef}}</a>
-						{{else}}
-							{{.PrettyRef}}
-						{{end}}
+					<b>{{if not $.CurWorkflow}}{{.WorkflowID}} {{end}}#{{.Index}}</b>
+					: {{$.locale.Tr "actions.runs.commit"}}
+					<a href="{{$.RepoLink}}/commit/{{.CommitSHA}}">{{ShortSha .CommitSHA}}</a>
+					{{$.locale.Tr "actions.runs.pushed_by"}}
+					<a href="{{.TriggerUser.HomeLink}}">{{.TriggerUser.GetDisplayName}}</a>
```

Comment:
I made a simple demo like this layout, and the spaces do not seem to work.

https://jsfiddle.net/0zuk3n9L/

![image](https://github.com/go-gitea/gitea/assets/2114189/14f645a9-b2f2-4e8b-a97f-9eb2bea0412d)

Reply:
But the space after ":" is there. For the other links we actually have margin to provide the space:

<img width="434" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/12ce5e5f-b41a-42bd-a665-71ad34d72c2a">

Reply:
Ah .... that's quite counterintuitive when reading the code.